config/development.json

```diff
@@ -138,6 +138,7 @@
 		"phone_signup": true,
 		"login": true,
 
+		"rubberband-scroll-disable": true,
```

Comment:
Well, for now I would start it with the app only and then we can change our minds